doc/tutorials/enable_s5.rst

```diff
@@ -1,374 +1,131 @@
-      ``/dev/ttySn``. For the User VM with the lowest VM ID, the communication
-      vUART id should be 1. For other User VMs, the vUART (id is 1) should be
-      configured as invalid; the communication vUART id should be 2 or higher.
-
-2. Build the Lifecycle Manager daemon, ``life_mngr``:
-
-   .. code-block:: none
-
-      cd acrn-hypervisor
-      make life_mngr
-
-#. For the Service VM, LaaG VM, and RT-Linux VM, run the Lifecycle Manager
-   daemon:
-
-   a. Copy ``life_mngr.conf``, ``s5_trigger_linux.py``, ``life_mngr``,
-      and ``life_mngr.service`` into the Service VM and User VMs. These commands
-      assume you have a network connection between the development computer and
-      target. You can also use a USB stick to transfer files.
-
-      .. code-block:: none
-
-         scp build/misc/services/s5_trigger_linux.py root@<target board address>:~/
-         scp build/misc/services/life_mngr root@<target board address>:/usr/bin/
-         scp build/misc/services/life_mngr.conf root@<target board address>:/etc/life_mngr/
-         scp build/misc/services/life_mngr.service root@<target board address>:/lib/systemd/system/
-
-   #. Copy ``user_vm_shutdown.py`` into the Service VM.
-
-      .. code-block:: none
-
-         scp misc/services/life_mngr/user_vm_shutdown.py root@<target board address>:~/
-
-   #. Edit options in ``/etc/life_mngr/life_mngr.conf`` in the Service VM.
-
-      .. code-block:: none
-
-         VM_TYPE=service_vm
-         VM_NAME=Service_VM
-         DEV_NAME=tty:/dev/ttyS8,/dev/ttyS9,/dev/ttyS10,/dev/ttyS11,/dev/ttyS12,/dev/ttyS13,/dev/ttyS14
-         ALLOW_TRIGGER_S5=/dev/ttySn
-
-      .. note:: The mapping between User VM ID and communication serial device
-         name (``/dev/ttySn``) is in the :file:`/etc/serial.conf`. If
-         ``/dev/ttySn`` is configured in the  ``ALLOW_TRIGGER_S5``, this means
-         system shutdown is allowed to be triggered in the corresponding User
-         VM.
-
-   #. Edit options in ``/etc/life_mngr/life_mngr.conf`` in the User VM.
-
-      .. code-block:: none
+#########
 
-         VM_TYPE=user_vm
-         VM_NAME=<User VM name>
-         DEV_NAME=tty:/dev/ttyS1
-         #ALLOW_TRIGGER_S5=/dev/ttySn
-
-      .. note:: The User VM name in this configuration file should be
-         consistent with the VM name in the launch script for the Post-launched
-         User VM or the VM name which is specified in the hypervisor scenario
-         XML for the Pre-launched User VM.
-
-   #. Use the following commands to enable ``life_mngr.service`` and restart the Service VM and User VMs.
+About System S5 Support
+***********************
 
-      .. code-block:: none
+S5 refers to the ACPI “soft off” system state. ACRN system S5 support enables
+you to gracefully shut down or reset the whole system when multiple VMs are
+running. This is done by requesting and waiting for all pre-launched and
+post-launched VMs to gracefully shut down themselves before the Service VM
```

Comment:
For my own education, shouldn't we say "shut themselves down" instead?

Reply:
Yes, I'll change that.